src/Pester.Runtime.psm1

```diff
@@ -175,6 +175,10 @@ function New-Block {
     $block = $null
     $previousBlock = $state.CurrentBlock
 
+    if (-not $PSBoundParameters.ContainsKey('StartLine')) {
+        $StartLine = $ScriptBlock.StartPosition.StartLine
```

Comment:
I think this should be `$MyInvocation.ScriptLineNumber` so we use the location of New-Block the same way we would use the location of Describe. If someone implements their own e.g. MyBlock they will get incorrect value from the start, showing them that they did something wrong, instead of mostly correct value which will break only of they move their scriptblock on the next line (which is the bug we were solving by this).

Reply:
I see your point. In that case. We should probably make it the default value for New-ParameterizedTest as well to cover tests using it directly (none using it with line filter atm, but maybe later). Adding a commit now

Reply:
Perfect!